VK/VK/Animations/Session/NetworkService.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  NetworkService.swift
+//  VK
+//
+//  Created by Алена Панченко on 22.11.2022.
+//
+
+import UIKit
+
+/// Сетевые запросы
+final class NetworkService {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let recommendationsText = "/recommendations"
+        static let uRLComponentsSchemeName = "https"
+        static let uRLComponentsHostName = "api.vk.com"
+        static let getFriendsDescription = "/method/friends.get"
+        static let getAllPhotosDescription = "/method/photos.getAll"
+        static let getGroupsDescription = "/method/groups.get"
+        static let searchGroupsDescription = "/method/groups.search"
+        static let queryItemTokenName = "access_token"
+        static let queryItemfieldsName = "fields"
+        static let queryItemVName = "v"
+        static let queryItemLandName = "land"
+        static let queryItemExtendedName = "extended"
+        static let queryItemQName = "q"
+        static let queryItemOwner_id = "owner_id"
+        static let queryItemLandValue = "ru"
+        static let queryItemVersionValue = "5.131"
+        static let queryItemExtendedValue = "1"
+        static let queryItemfieldsValue = "nickname"
+        static let queryItemQValue = "Bears"
+    }
+
+    enum MethodKind: CustomStringConvertible {
```

Comment:
```suggestion
// MARK: - Types

    enum MethodKind: CustomStringConvertible {
```
кажется у этого энама тоже должна быть марка